data/inline-histogram.go

```diff
@@ -180,17 +181,20 @@ func (ih *InlineHistogram) String() string {
 	return s
 }
 
-func (ih *InlineHistogram) HistogramToEmpiricalString() string {
-	s := fmt.Sprintf("InlineHistogram:\nBinCount: %v\nObservations: %v\nMin: %f\nMax: %f\nMean: %f\n", len(ih.bins), ih.pm.sampleSize, ih.pm.min, ih.pm.max, ih.pm.GetMean())
-	s += "Bin Start"
-	for idx, _ := range ih.bins {
-		s += fmt.Sprintf("%f, ", ih.minValue+(ih.binWidth*float64(idx)))
+func (ih *InlineHistogram) Bins() []float64 {
+	binStarts := []float64{}
+	for i, _ := range ih.bins {
+		binStarts = append(binStarts, float64(i))
 	}
-	s += "\nBin Count:"
+	return binStarts
+}
+
+func (ih *InlineHistogram) BinCounts() []float64 {
+	binCounts := []float64{}
```

Comment:
why not just return bins?

Reply:
I think the answer depends on your response to the above comment. I might misunderstand what the index of ih.bins represents.

Reply:
it is the count of values at a range the size of bin width on the real number line. to calculate the position use the min value + index)*binwidth  (assuming index starts at zero).